content/scalers/apache-kafka-topic.md

```diff
@@ -19,7 +19,7 @@ This specification describes the `kafka` trigger for Apache Kafka Topic.
   triggers:
```

Comment:
Don't forget to update the trigger specification section as well please

https://keda.sh/scalers/apache-kafka-topic/#trigger-specification

Reply:
Actually it's above but can't comment there